subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/BuildOperationsFixture.groovy

```diff
@@ -48,11 +48,15 @@ class BuildOperationsFixture {
     }
 
     @SuppressWarnings("GrUnnecessaryPublicModifier")
-    public <T extends BuildOperationType<?, ?>> List<BuildOperationRecord> roots() {
-        return operations.roots
+    public <T extends BuildOperationType<?, ?>> BuildOperationRecord root(Class<T> type, Spec<? super BuildOperationRecord> predicate = Specs.satisfyAll()) {
```

Comment:
Looks like the `predicate` is never supplied: I'd be tempted to leave it out until we need it.

Reply:
I originally left it out but added it for consistency with the other methods in this class.